samples/csharp/web/avatar/wwwroot/js/basic.js

```diff
@@ -0,0 +1,323 @@
+            })
+        } else {
+            throw new Error(`Failed fetching ICE token: ${response.status} ${response.statusText}`)
+        }
+    })
+}
+
+window.speak = () => {
+    document.getElementById('speak').disabled = true;
+    document.getElementById('stopSpeaking').disabled = false
+    document.getElementById('audio').muted = false
+    let spokenText = document.getElementById('spokenText').value
+    let ttsVoice = document.getElementById('ttsVoice').value
+    let personalVoiceSpeakerProfileID = document.getElementById('personalVoiceSpeakerProfileID').value
+    let spokenSsml = `<speak version='1.0' xmlns='http://www.w3.org/2001/10/synthesis' xmlns:mstts='http://www.w3.org/2001/mstts' xml:lang='en-US'><voice name='${ttsVoice}'><mstts:ttsembedding speakerProfileId='${personalVoiceSpeakerProfileID}'><mstts:leadingsilence-exact value='0'/>${htmlEncode(spokenText)}</mstts:ttsembedding></voice></speak>`
+    console.log("[" + (new Date()).toISOString() + "] Speak request sent.")
+
+    fetch('/api/speak', {
+        method: 'POST',
+        headers: {
+            'ClientId': clientId,
+            'Content-Type': 'application/ssml+xml'
+        },
+        body: spokenSsml
+    })
+    .then(response => {
+        document.getElementById('speak').disabled = false
+        document.getElementById('stopSpeaking').disabled = true
+        if (response.ok) {
+            response.text().then(text => {
+                console.log(`[${new Date().toISOString()}] Speech synthesized to speaker for text [ ${spokenText} ]. Result ID: ${text}`)
+            })
+        } else {
+            throw new Error(`[${new Date().toISOString()}] Unable to speak text. ${response.status} ${response.statusText}`)
+        }
+    })
+}
+
+window.stopSpeaking = () => {
+    document.getElementById('stopSpeaking').disabled = true
+
+    fetch('/api/stopSpeaking', {
+        method: 'POST',
+        headers: {
+            'ClientId': clientId
+        },
+        body: ''
+    })
+    .then(response => {
+        if (response.ok) {
+            console.log(`[${new Date().toISOString()}] Speaking stopped.`)
+            document.getElementById('speak').disabled = false
+            document.getElementById('stopSpeaking').disabled = false
+        } else {
+            throw new Error(`[${new Date().toISOString()}] Unable to stop speaking. ${response.status} ${response.statusText}`)
+        }
+    })
+}
+
+window.stopSession = () => {
+    document.getElementById('speak').disabled = true
+    document.getElementById('stopSpeaking').disabled = true
+    document.getElementById('stopSession').disabled = true
+
+    fetch('/api/disconnectAvatar', {
+        method: 'POST',
+        headers: {
+            'ClientId': clientId
+        },
+        body: ''
+    })
+}
+
+window.updataTransparentBackground = () => {
+    if (document.getElementById('transparentBackground').checked) {
+        document.body.background = './static/image/background.png'
```

Comment:
This path is incorrect.
It said the image cannot be found per my test.
Please fix.